src/System.Text.Json/src/System/Text/Json/Serialization/Converters/DefaultConverters.cs

```diff
@@ -58,6 +58,14 @@ internal static object Create(Type type)
             {
                 return new JsonValueConverterDateTimeOffset();
             }
+            if (type == typeof(JsonElement))
```

Comment:
Is it possible to change `Type.GetTypeCode(type)` here to include `JsonElement`/`object` so that it can fit in the above switch?

Reply:
Nope, `TypeCode` is an existing runtime enum for intrinsic types that is faster to check than type equivalency.